sdk/keyvault/keyvault-admin/samples/v4/javascript/README.md

```diff
@@ -16,7 +16,7 @@ These sample programs show how to use the JavaScript client libraries for Azure
 | ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
 | [accessControlHelloWorld.js][accesscontrolhelloworld] | Uses an AccessControlClient to list, create, and assign roles to users.                               |
 | [backupRestoreHelloWorld.js][backuprestorehelloworld] | Uses a BackupClient to backup and fully restore an Azure Key Vault using Azure Storage Blob.          |
-| [backupSelectiveRestore.js][backupselectiverestore]   | Uses a BackupClient to backup and restore a specific key in Azure Key Vault using Azure Storage Blob. |
```

Comment:
You might want to use the actual client class names, e.g. `KeyVaultBackupClient`. Same for AccessControlClient.